FoldingTabBar/View/FoldingTabBar/YALFoldingTabBar.m

```diff
@@ -621,13 +628,6 @@ - (void)animateCenterButtonCollapse {
 
 #pragma mark - Mutators
 
-- (void)setTabBarColor:(UIColor *)tabBarColor {
```

Comment:
@mmrmmlrr please revisit this method once more. Maybe we should keep this code as well. This changes were added intentionally in #26 and as I remember it was connected with this issue #23 
Thanks.